src/applyexp.jl

```diff
@@ -18,6 +18,7 @@ end
 
 struct ApplyExpInfo
   numops::Int
+  converged::Int
```

Comment:
I'm not a big fan of just using integers for this kind of thing. Is this either going to indicate that it is converged or not? If so we could use `Bool`, which would give more information.

Reply:
I would have done as you asked, where it not that the solver that the exponentiate solver returns an inbuilt info struct at tdvp.jl line 12, which contains `converged::Int ` I chose to implement all other info structs to mirror this choice, under the rationale that at least all the info structs we use from different solvers should be as similar as possible.

Reply:
I see, in `KrylovKit.jl` they use an integer value since some solvers return multiple solutions (like multiple eigenvectors) and `info.converged` indicates how many solutions were converged (https://jutho.github.io/KrylovKit.jl/stable/man/intro/#KrylovKit.ConvergenceInfo).

I guess we could follow a similar convention, @emstoudenmire what do you think?